docs-ref-services/latest/servicebus.md

```diff
@@ -15,402 +15,37 @@ ms.service: service-bus
-
-msg = Message(b'Hello World!')
-sbs.send_topic_message('taskdiscussion', msg)
-```
-
-The **send\_topic\_message_batch** method can be used to send  several messages at once:
-
-```python
-from azure.servicebus.control_client import Message
-
-msg1 = Message(b'Hello World!')
-msg2 = Message(b'Hello World again!')
-sbs.send_topic_message_batch('taskdiscussion', [msg1, msg2])
-```
-
-Please consider that in Python 3 a str message will be utf-8 encoded and you should have to manage your encoding yourself in Python 2.
-
-A client can then create a subscription and start consuming messages by calling the **create\_subscription** method followed by the **receive\_subscription\_message** method. Please note that any messages sent before the subscription is created will not be received.
-
-```python
-from azure.servicebus.control_client import Message
-
-sbs.create_subscription('taskdiscussion', 'client1')
-msg = Message('Hello World!')
-sbs.send_topic_message('taskdiscussion', msg)
-msg = sbs.receive_subscription_message('taskdiscussion', 'client1')
-```
-
-### Event Hub
-
-Event Hubs enable the collection of event streams at high throughput, from a diverse set of devices and services.
-
-The **create\_event\_hub** method can be used to create an event hub:
+## Libraries for data access
 
-```python
-sbs.create_event_hub('myhub')
-```
+The latest version of the Azure Servicebus libraries is version 7.x.x. We highly recommend using version 7.x.x for new applications.
 
-To send an event:
+To update existing applications to version 7.x.x, please follow the [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/migration_guide.md).
 
-```python
-sbs.send_event('myhub', '{ "DeviceId":"dev-01", "Temperature":"37.0" }')
-```
+### Version 7.x.x
 
-The event content is the event message or JSON-encoded string that contains multiple messages.
+To send and receive messages from an Azure Service Bus queue, topic or subscription, you would use the latest version of the `azure-servicebus`. This also allows to manage your Azure Service Bus resources like queues, topics, subscriptions and rules, but not the namespace itself.
 
-### Advanced features
+| Library | Reference | Samples | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    [azure-servicebus](https://pypi.org/project/azure-servicebus/)    |    [Reference](https://docs.microsoft.com/python/api/overview/azure/servicebus-readme?view=azure-python)    |    [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples)   |    [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus)    |
```

Comment:
Consider adding "v7", otherwise the link texts for this and the next table would look exactly the same since it is the same package.
Also, consider having more thank single words in link texts as it helps with accessibility. Screenreader users use this feature of looping through all links on the page and it is the link text that gets read out. Having more context in link text helps to differentiate between links

```suggestion
|    [azure-servicebus v7](https://pypi.org/project/azure-servicebus/)    |    [API Reference for azure-servicebus v7](https://docs.microsoft.com/python/api/overview/azure/servicebus-readme?view=azure-python)    |    [Samples for azure-servicebus v7](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples)   |    [Source code for azure-servicebus v7](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus)    |
```